### PR TITLE
Notify that a lobby is full when you watching a replay

### LINKED ISSUE #2836: The client does not notify that a lobby is full when you watching a replay
**To Reproduce**
1. Allow watching replays while in a game via `Settings -> Forget Alliance Forever`
2. Host a game or join to any game
3. Watch any replay
4. Wait when a lobby is full

**Log or error message**
No need

**Expected behavior**
Notify an user that the lobby is full

**Screenshots**
No need

**OS**
Windows

**Client Version**
2022.10.0-alpha-2

### PR DESCRIPTION
**Pull request by Marc-Spector**
Closes #2836

### CODE REVIEW

**Comment by Brutus5000** [comment]
Wouldn't it be easier to remember the process ids, when we launch a game of replay instead of fiddling around with window titles?

**Comment by Marc-Spector** [comment]
No need to remember process ids of game replay. It is enough to know the process id, where the lobby is located.

File: src/main/java/com/faforever/client/fx/PlatformService.java [hunk 1]
```diff
@@ -154,13 +187,21 @@ public void startFlashingWindow(String windowTitle) {
     User32.INSTANCE.FlashWindowEx(flashwinfo);
   }
 
-
   public void stopFlashingWindow(String windowTitle) {
+    startFlashingWindow(windowTitle, null);
+  }
```
Comment (Sheikah45) [inline src/main/java/com/faforever/client/fx/PlatformService.java]: is this supposed to be calling stopFlashingWindow?

**Review by Sheikah45 — CHANGES_REQUESTED** [review]

File: src/main/java/com/faforever/client/fx/PlatformService.java (hunk 1 above)
Comment (Marc-Spector) [reply src/main/java/com/faforever/client/fx/PlatformService.java]: Oh, good catch

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**